File: src/main/java/com/galaxy/Restaurantinformationsystem/RestaurantInformationSystemApplication.java
package com.galaxy.Restaurantinformationsystem;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class RestaurantInformationSystemApplication {


    public static void main(String[] args) {
        SpringApplication.run(RestaurantInformationSystemApplication.class, args);
    }

}

File: src/main/java/com/galaxy/restaurant/information/system/RestaurantInformationSystemApplication.kt
package com.galaxy.restaurant.information.system

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@EnableJpaAuditing
@SpringBootApplication
class RestaurantInformationSystemApplication

fun main(args: Array<String>) {
    runApplication<RestaurantInformationSystemApplication>(*args)
}
